Loop/Managers/DeviceDataManager.swift

```diff
@@ -468,20 +504,20 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate {
             }
         }
     }
-
+    
     var transmitterID: String? {
         didSet {
             if transmitterID?.characters.count != 6 {
                 transmitterID = nil
             }
-
+            
             switch (transmitterState, transmitterID) {
             case (.NeedsConfiguration, let transmitterID?):
                 transmitterState = .Ready(Transmitter(
                     ID: transmitterID,
                     startTimeInterval: NSUserDefaults.standardUserDefaults().transmitterStartTime,
                     passiveModeEnabled: true
-                ))
+                    ))
```

Comment:
interesting, but it re-indented it even when code around it didn't change? Not a big deal. I think Xcode needs to get with it.